app/controllers/admins_controller.rb

```diff
@@ -110,7 +107,7 @@ class UserSearch
 
     validate :any_searchfield_present?
 
-    def initialize(attributes={})
+    def initialize(attributes = {})
```

Comment:
https://github.com/styleguide/ruby 
Github styleguide recomends it. Anyway, trying to get around each devs taste will be futile, you'll have to admit this :)